var/spack/repos/builtin/packages/hdf5/package.py

```diff
@@ -188,35 +205,32 @@ def check_install(self):
             # TODO: Automate these path and library settings
             cc('-c', "-I%s" % join_path(spec.prefix, "include"), "check.c")
             cc('-o', "check", "check.o",
-               "-L%s" % join_path(spec.prefix, "lib"),
-               "-L%s" % join_path(spec.prefix, "lib64"),
```

Comment:
again, two `libs` are needed here because on some platforms it got installed into `lib64`. If not sure, don't delete lines.